src/components/layouts/Header.tsx

```diff
@@ -0,0 +1,31 @@
+'use client'
+
+import { Box, Stack, Typography } from '@mui/material'
+import { PrimaryBtn } from '../buttons/PrimaryBtn'
+import { Add } from '@mui/icons-material'
+import { AppMargin, SizeofAppMargin } from '@/hoc/AppMargin'
+import store from '@/redux/store'
+import { setShowModal } from '@/redux/features/createTaskSlice'
+
+export const Header = () => {
+  return (
+    <Box
+      sx={{
+        border: (theme) => `1px solid ${theme.color.borders.borderDisabled}`,
+      }}
+    >
+      <AppMargin size={SizeofAppMargin.LARGE} ptb="18.5px">
+        <Stack direction="row" alignItems="center" justifyContent="space-between" sx={{}}>
+          <Typography variant="lg">Tasks</Typography>
+          <PrimaryBtn
+            startIcon={<Add />}
+            buttonText={'New Task'}
```

Comment:
@aatbip extra curly brace

Reply:
removed in 92a681fbf84999366eace9a5b7d948254a06118f commit